packages/concerto-core/lib/decoratormanager.js

```diff
@@ -454,26 +620,26 @@ class DecoratorManager {
      */
     static executeNamespaceCommand(model, command) {
         const { target, decorator, type } = command;
-
         if (Object.keys(target).length === 2 && target.namespace) {
             const { name } = ModelUtil.parseNamespace( model.namespace );
-            // should we just compare with namespace??
-            if(this.falsyOrEqual(target.namespace, [model.namespace,name])) {
+            if(this.falsyOrEqual(target.namespace, [model.namespace, name])) {
                 this.applyDecorator(model, type, decorator);
             }
         }
     }
 
     /**
-     * Executes a Command against a ClassDeclaration, adding
-     * decorators to the ClassDeclaration, or its properties, as required.
+     * Executes a Command against a Declaration, adding
+     * decorators to the Declaration, or its properties, as required.
      * @param {string} namespace the namespace for the declaration
      * @param {*} declaration the class declaration
      * @param {*} command the Command object from the dcs
-     * @param {boolean} [enableDcsNamespaceTarget] - flag to control applying namespace targeted decorators on top of the namespace instead of all declarations in that namespace
+     * @param {*} property the property
```

Comment:
Looks like property is now an optional argument? Would be good to document that.

Reply:
Yes corrected